ensembl_dbcopy/forms.py

```diff
@@ -60,21 +74,24 @@ class Meta:
     user = forms.CharField(
         widget=forms.HiddenInput())
 
+    def _cleaned_text_field(self, field_name):
```

Comment:
I would personally have overriden CharField class with a "TrimmedCharField" with a overriden clean method
https://docs.djangoproject.com/en/2.2/howto/custom-model-fields/
For the regexp part, you can use: dedicated validator https://docs.djangoproject.com/en/2.2/ref/validators/#regexvalidator

Reply:
Good idea, I'll change that.